autoware_lanelet2_extension/CMakeLists.txt

```diff
@@ -2,6 +2,22 @@ cmake_minimum_required(VERSION 3.14)
 project(autoware_lanelet2_extension)
 
 find_package(autoware_cmake REQUIRED)
+
+find_package(TinyXML2 CONFIG QUIET)
+if(NOT TinyXML2_FOUND)
+  find_path(TINYXML2_INCLUDE_DIR NAMES tinyxml2.h)
+  find_library(TINYXML2_LIBRARY tinyxml2)
+  include(FindPackageHandleStandardArgs)
+  find_package_handle_standard_args(TinyXML2 DEFAULT_MSG TINYXML2_LIBRARY TINYXML2_INCLUDE_DIR)
+  mark_as_advanced(TINYXML2_INCLUDE_DIR TINYXML2_LIBRARY)
+  if(NOT TARGET tinyxml2::tinyxml2)
+    add_library(tinyxml2::tinyxml2 UNKNOWN IMPORTED)
+    set_property(TARGET tinyxml2::tinyxml2 PROPERTY IMPORTED_LOCATION ${TINYXML2_LIBRARY})
+    set_property(TARGET tinyxml2::tinyxml2 PROPERTY INTERFACE_INCLUDE_DIRECTORIES ${TINYXML2_INCLUDE_DIR})
+    list(APPEND TinyXML2_TARGETS tinyxml2::tinyxml2)
+  endif()
+endif()
+
```

Comment:
It looked to me like the `tinyxml2` isn’t used at all in this PR. Do we actually need it?

Reply:
https://github.com/autowarefoundation/autoware_lanelet2_extension/actions/runs/13588262489/job/37988013975?pr=47#step:6:2502

Reply:
@youtalk Actually I struggled to solve this tinyxm2 issue last month, still could not find any ways haha